opentelemetry-sdk/src/metrics/periodic_reader.rs

```diff
@@ -402,8 +412,13 @@ impl PeriodicReaderInner {
     }
 
     fn shutdown(&self) -> MetricResult<()> {
-        if self.is_shutdown.load(std::sync::atomic::Ordering::Relaxed) {
-            return Err(MetricError::Other("Reader is already shut down".into()));
+        if self
+            .is_shutdown
```

Comment:
We should rename this variable from `is_shutdown` to `shutdown_invoked` to reflect its true meaning.